extensions/chinese-dictionary/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "chinese-dictionary",
+  "title": "Chinese Dictionary",
+  "description": "Chinese Dictionary",
+  "icon": "icon.png",
+  "author": "CofCat",
+  "categories": [
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "chinese-dictionary",
+      "title": "Chinese Dictionary",
+      "description": "Find the textual definition",
```

Comment:
You could use `Find Textual Definition` as the title. Your command name is a duplicate of the extension name here. Using a more specific name allows users to reach our command faster.

Reply:
Thanks for the suggestion!